pkg/asset/manifests/operators.go

```diff
@@ -225,3 +227,12 @@ func applyTemplateData(template *template.Template, templateData interface{}) []
 	}
 	return buf.Bytes()
 }
+
+// Load returns the manifests asset from disk.
+func (m *Manifests) Load(p asset.FilePatternFetcher) (found bool, err error) {
+	files, found, err := p(fmt.Sprintf("%s/*", manifestDir))
```

Comment:
I am not sure that this will match on operating systems where the path separator is not `/`.